packages/wrangler/src/index.tsx

```diff
@@ -409,13 +411,17 @@ export async function main(argv: string[]): Promise<void> {
                 compilerOptions: {
                   target: "es2021",
                   module: "es2022",
+                  lib: ["es2021"],
                   moduleResolution: "node",
                   allowJs: true,
+                  esModuleInterop: true,
                   allowSyntheticDefaultImports: true,
                   isolatedModules: true,
-                  noEmit: true,
-                  lib: ["es2021"],
                   resolveJsonModule: true,
+                  forceConsistentCasingInFileNames: true,
+                  skipLibCheck: true,
+                  noEmit: true,
+                  strict: true,
```

Comment:
These changes don't seem to be relevant here?

Reply:
Oh hmm I thought I'd put this in another branch. I'll remove it from here.